src/components/v2/Slider/index.tsx

```diff
@@ -1,20 +1,29 @@
 /** @jsxImportSource @emotion/react */
 import React, { useState } from 'react';
 import MaterialSlider from '@mui/material/Slider';
+import Box from '@mui/material/Box';
+import { SliderTypeMap } from '@mui/material/Slider/Slider';
+
+import { Tooltip } from '../Tooltip';
 import { useStyles } from './styles';
 
 export interface ISliderProps {
+  value?: number;
+  /* value has to be provided for disabling slider changes */
+  isDisabled?: boolean;
   defaultValue: number;
   mark: number;
   step: number;
   ariaLabel: string;
   min: number;
   max: number;
   onChange: (value: number | number[]) => void;
-  isDisabled?: boolean;
+  trackTooltip?: string;
+  markTooltip?: string;
```

Comment:
Let's get rid of `defaultValue` then, since on our case the slider is never updated manually.

Reply:
it should be just progressBar component then

Reply:
We don't have any slider in the designs, so yeah it is progress bar :)

Reply:
done